### DISCUSSION TITLE
Visualization for completing all habits each day

### DISCUSSION THREAD

**Discussion by ganorberg** [iSoron/uhabits#552] [Feature Requests]
I'm loving the app so far, so thank you to everyone who has contributed to its success! 

Please correct me if I'm wrong, but I don't see any visualizations that track whether you have completed all of your habits for a particular day. For example, this photo shows how Productive has a calendar view with green circles filled in to show full completion, along with some aggregate stats: https://thesweetsetup.com/wp-content/uploads/2018/12/Productive-Habit-1.jpeg

I apologize if there is an issue for this already -- I scanned the existing issues and did a search for previous issues, but I didn't see this idea specifically. This was the closest relevant issue I could find: https://github.com/iSoron/uhabits/issues/521

I've never worked on an Android app before, but I'd be happy to learn and help build this into your app if you think it's a good idea and feasible. I'd also be willing to work closely with a designer to figure out the UI/UX and pair program with an Android developer to build it out.

**Comment by hiqua** [comment]
A few problems I see now with such a feature: what do you do when you add a new habit? Do you remember the time of creation to avoid checking whether it was completed before creation? What about archived / unarchived habits?

A workaround for now would be to have a habit "all other habits completed" which you would check manually.
